catalog/ui/search-ui/standard/src/main/webapp/js/view/MetacardDetail.view.js

```diff
@@ -22,23 +22,32 @@ define([
         'cometdinit',
         'text!templates/metacard.handlebars',
         'js/view/Modal',
-        'text!templates/metacardActionModal.handlebars'
+        'text!templates/metacardActionModal.handlebars',
+        'text!templates/metacardNearbyLocation.handlebars',
+        'js/view/NearbyLocation.view',
+        'js/model/NearbyLocation'
     ],
-    function ($, _, Marionette, ich, dir, maptype, wreqr, Cometd, metacardTemplate, Modal, metacardActionTemplate) {
+    function ($, _, Marionette, ich, dir, maptype, wreqr, Cometd, metacardTemplate, Modal, metacardActionTemplate, metacardNearbyLocationTemplate, NearbyLocationView, NearbyLocation) {
         "use strict";
 
         var Metacard = {};
 
+        var nearbyChecked = false;
+
         ich.addTemplate('metacardTemplate', metacardTemplate);
         ich.addTemplate('metacardActionTemplate', metacardActionTemplate);
+        ich.addTemplate('metacardNearbyLocationTemplate', metacardNearbyLocationTemplate);
```

Comment:
We typically do the adding of the Template in the relevant js file, so in this case you might want to consider moving this to ```NearbyLocation.view.js```.